examples/player/simid_player.js

```diff
@@ -477,6 +495,7 @@ class SimidPlayer {
       const closeMessage = {
         'code': reason,
       }
+      this.setCreativeControlsState_(/* controlState= */ DISABLED);
```

Comment:
Could this be missed if there were a skip ad or fatal error?  How about putting this code in destroyIframeAndResumeContent_

Reply:
Well in our sample creative we don't show demonstrate a skip ad or fatal error functionality via the creative (would seem counter intuitive if the creative skipped itself). But if you want me to move it for versatility then I can